ui/hooks/objects.ts

```diff
@@ -66,3 +74,29 @@ export function useGetObject<T extends FluxObject>(
   }
   return response;
 }
+
+type Res = { objects: FluxObject[]; errors: ListError[] };
+
+export function useListObjects<T extends FluxObject>(
```

Comment:
Didn't we say something on… Friday was it? About how there can be objects in different clusters with the same name, so we need to make sure this only queries the right cluster?

Reply:
AH, so it was not implemented yet? OK, I will look into it.

Reply:
I had a sudden realization: now that this feature is here, we will be able to use it in `SourceDetails` for populating the relevant Automation list.